processor/src/stack/inputs.rs

```diff
@@ -0,0 +1,66 @@
+use super::{vec, Felt, Vec};
+use core::slice;
+
+// STACK INPUTS
+// ================================================================================================
+
+/// Initial state of the stack to support program execution.
+///
+/// The program execution expects the inputs to be a stack on the VM, and it will be stored in
+/// reversed order on this struct.
+#[derive(Clone, Debug, Default)]
+pub struct StackInputs {
+    values: Vec<Felt>,
+}
+
+impl StackInputs {
+    // CONSTRUCTORS
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns `[StackInputs]` from a list of values, reversing them into a stack.
+    pub fn new(mut values: Vec<Felt>) -> Self {
+        values.reverse();
+        Self { values }
+    }
+
+    /// Returns `[StackInputs]` without values.
+    pub const fn empty() -> Self {
+        Self { values: vec![] }
+    }
+
+    // PUBLIC ACCESSORS
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns the initial stack values in stack/reversed order.
+    pub fn values(&self) -> &[Felt] {
+        &self.values
+    }
+}
+
+impl<F> FromIterator<F> for StackInputs
+where
```

Comment:
Should we also have a `TryFrom` for a list of `u64` values?

Reply:
This is actually an open question from the previous PR: why do we use `try_from` for `u64` for `Felt` when the modulus reduction is always performed upon a new felt? The idea is to give feedback to the user if he inputs something greater than the modulus? Shouldn't we just wrap that number?

Reply:
The idea is that if a user provides a `u64` greater than modulus it is very likely an error and could lead to unexpected behavior (i.e., user provides `modulus + 1` as input, but it gets silently converted to `1`).

Reply:
Created a `pub fn try_from_values<I>(iter: I) -> Result<Self, InputError>` and moved `FromIterator<impl Into<Felt>>` to `test` scope so the user will unmistakably only call `try_from_values`

Reply:
Update: I removed the `FromIterator` so there will be only one way for the user to create inputs: the right way